db/migrate/20220705134148_remove_ref_from_comments.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRefFromComments < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
- [ ] When running the migration I am getting the following error please fix that 👇 the next reviewer will make sure you tests are passing after you fix the issue 😉 

![image](https://user-images.githubusercontent.com/75126481/177961503-20ac65cb-faf2-42d1-9756-9b39474b5b2b.png)